book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ] Let's create a method for adding rentals. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Check out this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) as a reference.

Reply:
Oh, it is true! I missed that method, thanks!